### DISCUSSION TITLE
Example repository with Git translation

### DISCUSSION THREAD

**Discussion by popuguytheparrot** [facebook/docusaurus#4484] [Q&A]
Hello everyone, I'm trying to add localization via git, but I don't understand what to do. I have read the guide several times, but I have problems not understanding what to do in the end. How to localize routing, home page, sidebar, navbar, md files. Please share the repository, where you can clearly see

**Comment by slorber** [comment]
Hi,

Sorry that you can't make this work. 
I plan to add a repo including translations soon!

In the meantime, can you explain exactly what did not work, what did you attempt? 
What is not clear in the tutorial exactly?
If you had a git repo with your attempt I could tell you what you did wrong.

**Reply by popuguytheparrot** [reply]
Hi. Thank you for answer.

https://github.com/effector/effector

i tried, but don't commit changes yet.

In the tutorial i don't understand how translate jsx pages, how to make localized routes.
I tried to translate the navigation but the translation didn't apply. (Yes, I used a cli)

**Reply by slorber** [reply]
sorry but if you want me to help you, I really need to see your code, and to understand what exactly you tried. Please quote specific parts of our doc that you don't understand so that we can discuss those.

At this point, I'm not even 100% sure that you are actually reading Docusaurrus v2 i18n doc as all this discussion is quite "abstract" 😅

**Reply by popuguytheparrot** [reply]
Yes, that's not right. I will commit and write here how I get it. Thank
